src/js/lib/monster.ui.js

```diff
@@ -475,9 +475,34 @@ define(function(require) {
 				options = _.merge({
 					search_contains: true,
 					width: '220px'
-				}, pOptions);
+				}, pOptions),
+				chosen;
 
 			$target.chosen(options);
+
+			if (!pOptions.tags) {
+				return;
+			}
+
+			chosen = $target.data('chosen');
```

Comment:
I think it would be better to change the name of this variable, to not to cause any confusion with the method name when a dev scans the code quickly.

Reply:
Agree, since `data('chosen')` effectively returns the Chosen instance, you can rename this variable to `instance`.

Reply:
awesome, than yuo guys